designs/bee/src/cup.mjs

```diff
@@ -7,12 +7,29 @@ export const cup = {
   hideDependencies: true,
   after: neckTie,
   options: {
+    //Bee's
     topDepth: { pct: 54, min: 50, max: 80, menu: 'fit' },
     bottomCupDepth: { pct: 8, min: 0, max: 20, menu: 'fit' },
     sideDepth: { pct: 20.6, min: 0, max: 30, menu: 'fit' },
     sideCurve: { pct: 0, min: -50, max: 50, menu: 'fit' },
     frontCurve: { pct: 0, min: -50, max: 50, menu: 'fit' },
     bellaGuide: { bool: false, menu: 'fit' },
+    //changed from Bella
+    backArmholeCurvature: 0.63,
+    backArmholePitchDepth: 0.35,
+    backArmholeSlant: 5,
+    frontArmholeCurvature: 0.63,
+    bustDartCurve: 1,
+    bustDartLength: 1,
+    waistDartLength: 1,
+    backHemSlope: 2.5,
+    backNeckCutout: 0.06,
+    //catergory changed from Bella
+    armholeDepth: { pct: 44, min: 38, max: 46, menu: 'advanced.bellaArmhole' },
```

Comment:
Options don't belong to a design. They belong to a part. So if you want to have the menu reference that, please use the design name + part name.

Reply:
This how they were labeled before the port. It's a prefix to the catergories they were in under Bella.